homeassistant/components/poolstation/entity.py

```diff
@@ -0,0 +1,40 @@
+"""Base class for Poolstation entity."""
+from __future__ import annotations
+
+from pypoolstation import Pool
+
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from . import PoolstationDataUpdateCoordinator
+from .const import DOMAIN
+
+MANUFACTURER = "MANUFACTURER: WORK IN PROGRESS"
+MODEL = "MODEL: WORK IN PROGRESS"
```

Comment:
:construction: Work in progress

Reply:
I forgot about this. I don't use this anymore, I couldn't find in the API any information about this.